fields/5-icca-no-take.json

```diff
@@ -1,10 +1,24 @@
 {
   "key": "icca-no-take",
```

Comment:
Do you want to keep this as the key for this question? This is the equivalent to the name of the database column for this value: when this is exported to GeoJSON this will be the key that refers to this property, e.g. it would be something like:

```json
{
  "properties": {
    "icca-no-take": "indigenous_people"
  }
}
```

Reply:
ah yes, thank you renamed.